src/main/java/util/Storage.java

```diff
@@ -0,0 +1,111 @@
+package util;
+
+import alanExceptions.AlanException;
+import alanExceptions.FileReadException;
+import alanExceptions.FileWriteException;
+import alanExceptions.SaveFileException;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * This class is used to read and write files.
+ */
+public class Storage {
+    private Path dirPath = Paths.get("data");
```

Comment:
I like how you made as many of the relevant fields private. Good coding practice. 👍